integration_tests/src/main/python/cache_test.py

```diff
@@ -64,7 +64,7 @@ def do_join(spark):
         cached = left.join(right, left.a == right.r_a, join_type).cache()
         cached.count() # populates cache
         return cached
-
+    enableVectorizedConf.update(allowNegativeDecimalConf)
```

Comment:
This actually changes the global `enableVectorizedConf`, so any test that happens to run in the same process after this one runs that also uses  `enableVectorizedConf` will get the modified version.  I don't think we want that.